### DISCUSSION TITLE
第6节课笔记(15班)

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#41] [笔记]
请在下方提交笔记链接

**Comment by bblgbr** [comment]
第六节课笔记：https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/06%20OpenCompass%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%AF%84%E6%B5%8B.md

**Comment by lizhunkg** [comment]
第六节课笔记：https://note.youdao.com/s/WZNSkaes

**Comment by dqyydq** [comment]
https://blog.csdn.net/qq_52410386/article/details/135759909?spm=1001.2014.3001.5501

**Comment by szhowardhuang** [comment]
笔记见附件，和基础作业做在一起
[OpenCompass介绍.pdf](https://github.com/InternLM/tutorial/files/14020060/OpenCompass.pdf)

**Comment by Mrguanglei** [comment]
https://mqf0lvyk616.feishu.cn/wiki/PrNZw01gyiIm6Uk02y4cvxZHn4b?from=from_copylink

**Comment by scchy** [comment]
第六节课笔记:
https://github.com/scchy/My_Learn/blob/master/LLM_prime/6_InternLM_opc_doc.md

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/dktv4kfn629so6qp?singleDoc# 《lesson6 OpenCompass 大模型评测笔记》

**Comment by sokolo05** [comment]
第六节课笔记：https://github.com/sokolo05/Scholar_PuYu/blob/main/02.%E8%AF%BE%E7%A8%8B%E7%AC%94%E8%AE%B0/%E7%AC%AC6%E8%8A%82_%E7%AC%94%E8%AE%B0.md

**Comment by ziptoam** [comment]
第六节课笔记
【有道云笔记】第六节课笔记与作业
https://note.youdao.com/s/QDJn106x

**Comment by woodx9** [comment]
请参考笔记相关部分[飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/AhJQw4CTAiTMcWkzK36cC7uenrf?from=from_copylink)

**Comment by IyGnahz** [comment]
笔记+作业 https://g6xuom3zbh.feishu.cn/docx/UcyXd0YUCoMDEKxuW55cvy08nnG?from=from_copylink

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/VgrijQEd

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/WRVuwVezHiVs4Zkoiizcd6n0nVh?from=from_copylink

**Comment by g29times** [comment]
https://novagen.notion.site/6-OpenCompass-d80ca87a81c340e289336cb6be6bf3fd?pvs=4
